README.md

```diff
@@ -66,11 +66,11 @@ pyenv/bin/python setup.py install
 
 ## Configuration
 
-A sample config file is stored in `pdp/config.yaml`. Copy this file to a new location and set an environment variable `PDP_CONFIG` to point to it.
+Configuration of the PDP is accomplished through a set of environment variables. A sample environment file is stored in `pdp/config.env`. This environment file can be sourced in before you run the pdp, included in a Docker deployment or used in any other flexible way.
 
 ```bash
-cp pdp/config.yaml ~/pdp_config.yaml
```

Comment:
Do lines 187 and 201 (pdp.yaml files for supervisord) also need to be updated?

Reply:
Yeah, we don't really use supervisord anymore, but I can make that more clear.